Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8-alpine3.6
```

Comment:
I think it would be better if you would use `golang:alpine` which points to the latest Go version.

Reply:
I like pinning of versions because you have a defined state that way. Without version tags people end up with different build environments depending on when they did a docker build.
I would vote for keeping it the way it is.

Reply:
That is a point which I didn't thought of. With that in mind, I'd also go your way.